bindings/java/iota-wallet-java/lib/src/main/java/org/iota/types/PreparedTransactionData.java

```diff
@@ -8,14 +8,14 @@
 public class PreparedTransactionData extends AbstractObject {
 
     private TransactionEssence essence;
-    private InputSigningData inputsData;
+    private InputSigningData inputsData[];
```

Comment:
```suggestion
    private InputSigningData[] inputsData;
```

Reply:
Its technically the same thing by the way :) but was on accident